src/main/java/com/nerodesk/takes/doc/TkRead.java

```diff
@@ -64,7 +66,15 @@
     public Response act(final Request req) throws IOException {
         final ByteArrayOutputStream baos = new ByteArrayOutputStream();
         new RqDoc(req, this.base).doc().read(baos);
-        return new RsWithBody(new ByteArrayInputStream(baos.toByteArray()));
+        return
+            new RsWithHeader(
```

Comment:
@dmzaytsev what happened with indentation here? just put this `new RsWithHeader` to the previous line and remove checkstyle suppression